include/audio/soundfont.h

```diff
@@ -0,0 +1,101 @@
+#ifndef AUDIO_SOUNDFONT_H
+#define AUDIO_SOUNDFONT_H
+
+#include "PR/ultratypes.h"
+
+struct EnvelopePoint;
+
+typedef struct AdpcmLoop {
+    /* 0x00 */ u32 start;
+    /* 0x04 */ u32 loopEnd; // numSamples into the sample where the loop ends
```

Comment:
To me, `loopEnd` sounds like an address, not a length in cycles, is that just me? `loopCycleLen`? `loopMaxCount`?

Reply:
Yeah, I see what you mean. I don't think it's quite a length/count since if the start pos is 10, and the end pos is 50, then I would picture the length/count to be 40, but in this case it would be the 50 stored in the variable.

What about `loopEndPos`? Is that clearer? Maybe `startPos` and sampleEndPos` could compliment that name in this struct?

Reply:
Oh I misunderstood what it was doing, if its something more like a position then it could probably be left named. I just misunderstood `numSamples` was a unit in the comment.

Reply:
I adjusted the comments a little, just to mention it's in terms of position. Were you find with the names as is?

Reply:
ah yeah, now that I know